terraform/per_account/deployable/lambda_cloudwatch_event_rule_forwarder.tf

```diff
@@ -23,20 +23,56 @@ resource "aws_lambda_function" "cloudwatch_event_rule_forwarder_lambda" {
   }
 }
 
-data "aws_sns_topic" "codepipeline_health_notification" {
-  name = "codepipeline-health-notification"
+resource "aws_lambda_permission" "cloudwatch_forwarder_events_euw1_sns_invoke" {
+  statement_id  = "CloudForwarderCodepipelineAllowExecutionFromSNS"
+  action        = "lambda:InvokeFunction"
+  function_name = aws_lambda_function.cloudwatch_event_rule_forwarder_euw1_lambda.function_name
+  principal     = "sns.amazonaws.com"
+  source_arn    = aws_sns_topic.euw1_cloudwatch_event_sns_topic.arn
+}
```

Comment:
Considering these changes plan successfully, I'm somewhat impressed that Terraform infers that this resource should be placed in `eu-west-1`, as there's no explicit `provider` argument to tell it to use `aws.eu-west-1`.

Reply:
Good point - it doesn't. I'll change that.